frontend/src/bundles/auth/store/actions.ts

```diff
@@ -36,6 +39,7 @@ const signIn = createAsyncThunk<
     void storage.set(StorageKey.TOKEN, token);
 
     await dispatch(loadUser());
+    return { token };
```

Comment:
why do we need a return token?

Reply:
Resolved in [aa800ed](https://github.com/BinaryStudioAcademy/bsa-winter-2022-2023-smartspend/pull/120/commits/aa800edc5e4cac6b991472363a64b9834428bccc)